.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Question
+about: Question regarding Alembic features
+title: ''
+labels: requires triage
+assignees: ''
+
+---
+
+**Describe you question**
```

Comment:
typo, "your"

Reply:
Good catch, I think it's also in sqlalchemy. Will correct